src/app/screens/BookDetail/components/BookInfo/styles.ts

```diff
@@ -0,0 +1,72 @@
+import { StyleSheet } from 'react-native';
+import { white, gray, blue, lightGray, red } from '@constants/colors';
+import { SHADOW } from '@constants/styleHelpers';
+
+export default StyleSheet.create({
+  bookInfo: {
+    ...SHADOW,
+    backgroundColor: white,
+    paddingHorizontal: 30,
+    paddingVertical: 20,
+    borderRadius: 5,
+    marginBottom: 10,
+    alignItems: 'center'
+  },
+  bookInfoInside: {
+    flexDirection: 'row',
+    marginBottom: 30
+  },
+  bookSummary: {
+    flex: 1
+  },
+  bookCover: {
+    width: 70,
+    height: 110,
+    marginRight: 20
+  },
+  bookTitle: {
+    fontSize: 24,
+    lineHeight: 28,
+    fontWeight: 'bold',
+    color: gray
+  },
+  bookStatus: {
+    color: red,
+    fontWeight: 'bold',
+    fontSize: 16,
+    lineHeight: 20
+  },
+  bookBasicAttribute: {
+    fontSize: 16,
+    lineHeight: 20,
```

Comment:
For the future, you can extract the common props:
```
const MEDIUM_FONT = { fontSize: 16, lineHeight: 20 };
...
bookStatus: { ...MEDIUM_FONT, ... },
bookBasicAttribute: { ...MEDIUM_FONT, ... },
...
```
Just for comment, no need to change it now.

Reply:
Thanks so much!